simdjson/__init__.pyi

```diff
@@ -0,0 +1,108 @@
+import json
+from typing import (
+    AbstractSet,
+    Any,
+    Dict,
+    Final,
+    Iterator,
+    List,
+    Literal,
+    MutableMapping,
+    NewType,
+    Optional,
+    Tuple,
+    Union,
+    ValuesView,
+)
+
+
+class Object(MutableMapping[Any, Any]):
+    def __delitem__(self, key: Any) -> None:
```

Comment:
Our object is not a MutableMapping - simdjson is read-only, so we don't support __delitem__ or __setitem__.

Since we're mapping JSON, not arbitrary dicts, we can be a lot more specific than `[Any, Any]`. Keys can only ever be strings, and values can only ever be one of the primitive types or our {Object, Array} proxies.

Reply:
Will `Union[Object,Array]` be true also when you do `get` and `at_pointer`? I mean, at some point, you will find a value that is an ordinary `float` or `boolean`, right?

Reply:
Ok, I applied your comments @TkTech . I will resolve this comment for the sake of 2nd round ;-)